library/Freckle/App/Database.hs

```diff
@@ -65,12 +70,64 @@ makePostgresPool = do
   makePostgresPoolWith conf
 
 runDB
-  :: (HasSqlPool app, MonadUnliftIO m, MonadReader app m)
+  :: ( HasSqlPool app
+     , Stats.HasStatsClient app
```

Comment:
We tend to do a "type un-qualified, functions qualified" thing, just like you would with `Text` or `HashMap`,

```hs
import Freckle.App.Stats (HasStatsClient)
import qualified Freckle.App.Stats as Stats
```